src/ocean/io/Console.d

```diff
@@ -498,16 +498,3 @@ static ~this()
     Cout.flush;
     Cerr.flush;
 }
-
-
-/******************************************************************************
-
- ******************************************************************************/
-
-debug (Console)
-{
-    void main()
-    {
-        Cout ("hello world").newline;
```

Comment:
I love the removal of old ununsed code, but it would be nice if these tests are not *lost* (I know, since they are never ran by anyone technically they are already lost, but still). I would add the tests as unittests when possible and when not as *integrationtest*s, if it's not too much trouble (i.e. only "saving" the low hanging fruit).